apps/web/src/app/article/[article_id]/article.module.css

```
@@ -0,0 +1,419 @@
+.bodySection {
+  background: #ffffff;
+  border: 1px solid #e0e4ea;
+  border-radius: 12px;
+  padding: 1.5rem;
+}
+
+.bodyContent {
+  color: #374151;
+  line-height: 1.75;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)